README.md

```diff
@@ -649,7 +649,16 @@ but if you need to do this manually, you can use the exposed functions for this:
 
 </details>
 
+# Unit testing with Jest
 
+The dapp-core library exposes bundles for both CommonJS and ESModules, however, in some enviornments, Jest might require manual mapping of the CommonJS output. To implement it, add the following snippet inside your jest config file.
```

Comment:
`The dapp-core library exposes bundles for both CommonJS and ESModules. In some environments, Jest might require manual mapping of the CommonJS output. To implement it, add the following snippet inside your Jest config file.`